cognite/client/_api/relationships.py

```diff
@@ -186,6 +229,8 @@ def list(
             labels (LabelFilter): Return only the resource matching the specified label constraints.
             limit (int): Maximum number of relationships to return. Defaults to 100. Set to -1, float("inf") or None
                 to return all items.
+            fetch_resources (bool): if true, will try to return the full resources referenced by the relationship in the
+                source and target fields.
```

Comment:
Maybe
```suggestion
                source_type and target_type fields.
```

Reply:
it is also based on the source_external_id and target_external_id

Reply:
Oh no, this seems to be correct, sorry!